boilerplate.js

```diff
@@ -143,6 +158,15 @@ async function install(context) {
       )
     )(currentPackage)
 
+    if (!includeDetox) {
+      delete newPackage.detox
+      delete newPackage.scripts["test:e2e"]
+      delete newPackage.scripts["build:e2e"]
+      delete newPackage.scripts["ci:build:e2e"]
+      delete newPackage.scripts["ci:test:e2e"]
+      delete newPackage.devDependencies.detox
+    }
```

Comment:
Can we just conditionally include them in the `package.json.ejs` instead? Will put an example below.

Reply:
Good point!